<h1 align="center">
    best-of-bioinformatics
    <br>
</h1>


## Contents

- [Tools for FASTA/FASTQ](#tools-for-fastafastq) _13 projects_
- [Tools for BED/GTF/GFF](#tools-for-bedgtfgff) _2 projects_
- [Tools for SAM/BAM](#tools-for-sambam) _10 projects_
- [Tools for VCF/BCF](#tools-for-vcfbcf) _5 projects_
- [Assembler](#assembler) _3 projects_
- [NGS Alignment](#ngs-alignment) _10 projects_
- [Variant Caller](#variant-caller) _8 projects_
- [Variant Annotation](#variant-annotation) _4 projects_
- [Structural Variation](#structural-variation) _9 projects_
- [MSI Detection](#msi-detection) _4 projects_
- [MHC/HLA](#mhchla) _4 projects_
- [Sequence Alignment](#sequence-alignment) _7 projects_
- [Visualization](#visualization) _5 projects_
- [Others](#others) _14 projects_

## Explanation
- ğŸ¥‡ğŸ¥ˆğŸ¥‰&nbsp; Combined project-quality score
- â­ï¸&nbsp; Star count from GitHub
- ğŸ£&nbsp; New project _(less than 12 months old)_
- ğŸ’¤&nbsp; Inactive project _(12 months no activity)_
- ğŸ’€&nbsp; Dead project _(36 months no activity)_
- ğŸ“ˆğŸ“‰&nbsp; Project is trending up or down
- â•&nbsp; Project was recently added
- ğŸ‘¨â€ğŸ’»&nbsp; Contributors count from GitHub
- ğŸ”€&nbsp; Fork count from GitHub
- ğŸ“‹&nbsp; Issue count from GitHub
- â±ï¸&nbsp; Last update timestamp on package manager
- ğŸ“¥&nbsp; Download count from package manager
- ğŸ“¦&nbsp; Number of dependent projects

<br>

## Tools for FASTA/FASTQ

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/shenwei356/seqkit">seqkit</a></b> (ğŸ¥‡18 Â·  â­ 590) - A cross-platform and ultrafast toolkit for FASTA/Q file manipulation in Golang.</summary>

- [GitHub](https://github.com/shenwei356/seqkit) (ğŸ‘¨â€ğŸ’» 5 Â· ğŸ”€ 87 Â· ğŸ“¥ 42K Â· ğŸ“‹ 150 - 5% open Â· â±ï¸ 31.01.2021):

	```
	git clone https://github.com/shenwei356/seqkit
	```
</details>
<details><summary><b><a href="https://github.com/mdshw5/pyfaidx">pyfaidx</a></b> (ğŸ¥‡18 Â·  â­ 300) - Efficient pythonic random access to fasta subsequences.</summary>

- [GitHub](https://github.com/mdshw5/pyfaidx) (ğŸ‘¨â€ğŸ’» 25 Â· ğŸ”€ 50 Â· ğŸ“¦ 440 Â· ğŸ“‹ 120 - 10% open Â· â±ï¸ 08.12.2020):

	```
	git clone https://github.com/mdshw5/pyfaidx
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/fastp">fastp</a></b> (ğŸ¥ˆ16 Â·  â­ 940) - An ultra-fast all-in-one FASTQ preprocessor..</summary>

- [GitHub](https://github.com/OpenGene/fastp) (ğŸ‘¨â€ğŸ’» 17 Â· ğŸ”€ 190 Â· ğŸ“‹ 290 - 50% open Â· â±ï¸ 14.10.2020):

	```
	git clone https://github.com/OpenGene/fastp
	```
</details>
<details><summary><b><a href="https://github.com/gmarcais/Jellyfish">Jellyfish</a></b> (ğŸ¥ˆ16 Â·  â­ 280) - A fast multi-threaded k-mer counter.</summary>

- [GitHub](https://github.com/gmarcais/Jellyfish) (ğŸ‘¨â€ğŸ’» 12 Â· ğŸ”€ 120 Â· ğŸ“¥ 84K Â· ğŸ“‹ 150 - 45% open Â· â±ï¸ 05.05.2020):

	```
	git clone https://github.com/gmarcais/Jellyfish
	```
</details>
<details><summary><b><a href="https://github.com/marcelm/cutadapt">cutadapt</a></b> (ğŸ¥ˆ15 Â·  â­ 340 Â· â•) - Cutadapt removes adapter sequences from sequencing reads.</summary>

- [GitHub](https://github.com/marcelm/cutadapt) (ğŸ‘¨â€ğŸ’» 26 Â· ğŸ”€ 98 Â· ğŸ“¦ 190 Â· ğŸ“‹ 460 - 4% open Â· â±ï¸ 02.02.2021):

	```
	git clone https://github.com/marcelm/cutadapt
	```
</details>
<details><summary><b><a href="https://github.com/lh3/seqtk">seqtk</a></b> (ğŸ¥‰14 Â·  â­ 780) - Toolkit for processing sequences in FASTA/Q formats.</summary>

- [GitHub](https://github.com/lh3/seqtk) (ğŸ‘¨â€ğŸ’» 8 Â· ğŸ”€ 230 Â· ğŸ“‹ 130 - 24% open Â· â±ï¸ 04.12.2020):

	```
	git clone https://github.com/lh3/seqtk
	```
</details>
<details><summary><b><a href="https://github.com/agordon/fastx_toolkit">fastx_toolkit</a></b> (ğŸ¥‰13 Â·  â­ 120 Â· ğŸ’€) - FASTA/FASTQ pre-processing programs.</summary>

- [GitHub](https://github.com/agordon/fastx_toolkit) (ğŸ‘¨â€ğŸ’» 3 Â· ğŸ”€ 52 Â· ğŸ“¥ 68K Â· ğŸ“‹ 18 - 38% open Â· â±ï¸ 14.08.2017):

	```
	git clone https://github.com/agordon/fastx_toolkit
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/MutScan">MutScan</a></b> (ğŸ¥‰13 Â·  â­ 110 Â· ğŸ’¤) - Detect and visualize target mutations by scanning FastQ files directly.</summary>

- [GitHub](https://github.com/OpenGene/MutScan) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 33 Â· ğŸ“¥ 7 Â· ğŸ“‹ 10 - 30% open Â· â±ï¸ 21.01.2020):

	```
	git clone https://github.com/OpenGene/MutScan
	```
</details>
<details><summary><b><a href="https://github.com/s-andrews/FastQC">FastQC</a></b> (ğŸ¥‰12 Â·  â­ 160) - A quality control analysis tool for high throughput sequencing data.</summary>

- [GitHub](https://github.com/s-andrews/FastQC) (ğŸ‘¨â€ğŸ’» 7 Â· ğŸ”€ 43 Â· ğŸ“‹ 65 - 32% open Â· â±ï¸ 13.01.2021):

	```
	git clone https://github.com/s-andrews/FastQC
	```
</details>
<details><summary><b><a href="https://github.com/Genetalks/gtz">gtz</a></b> (ğŸ¥‰11 Â·  â­ 140) - A high performance and compression ratio compressor for genomic data, powered by..</summary>

- [GitHub](https://github.com/Genetalks/gtz) (ğŸ‘¨â€ğŸ’» 17 Â· ğŸ”€ 36 Â· ğŸ“‹ 29 - 31% open Â· â±ï¸ 15.12.2020):

	```
	git clone https://github.com/Genetalks/gtz
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/repaq">repaq</a></b> (ğŸ¥‰11 Â·  â­ 82) - A fast lossless FASTQ compressor with ultra-high compression ratio.</summary>

- [GitHub](https://github.com/OpenGene/repaq) (ğŸ‘¨â€ğŸ’» 3 Â· ğŸ”€ 15 Â· ğŸ“‹ 18 - 33% open Â· â±ï¸ 14.06.2020):

	```
	git clone https://github.com/OpenGene/repaq
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/fastv">fastv</a></b> (ğŸ¥‰10 Â·  â­ 69 Â· ğŸ£) - An ultra-fast tool for identification of SARS-CoV-2 and other microbes from..</summary>

- [GitHub](https://github.com/OpenGene/fastv) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 11 Â· ğŸ“‹ 11 - 36% open Â· â±ï¸ 29.09.2020):

	```
	git clone https://github.com/OpenGene/fastv
	```
</details>
<details><summary><b><a href="https://github.com/shubhamchandak94/Spring">Spring</a></b> (ğŸ¥‰10 Â·  â­ 43) - FASTQ compression.</summary>

- [GitHub](https://github.com/shubhamchandak94/Spring) (ğŸ”€ 14 Â· â±ï¸ 14.11.2020):

	```
	git clone https://github.com/shubhamchandak94/Spring
	```
</details>
<br>

## Tools for BED/GTF/GFF

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/arq5x/bedtools2">bedtools2</a></b> (ğŸ¥‡22 Â·  â­ 650) - bedtools - the swiss army knife for genome arithmetic.</summary>

- [GitHub](https://github.com/arq5x/bedtools2) (ğŸ‘¨â€ğŸ’» 85 Â· ğŸ”€ 240 Â· ğŸ“¥ 360K Â· ğŸ“‹ 670 - 20% open Â· â±ï¸ 04.02.2021):

	```
	git clone https://github.com/arq5x/bedtools2
	```
</details>
<details><summary><b><a href="https://github.com/lh3/bedtk">bedtk</a></b> (ğŸ¥‰8 Â·  â­ 100) - A simple toolset for BED files (warning: CLI may change before bedtk becomes..</summary>

- [GitHub](https://github.com/lh3/bedtk) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 9 Â· â±ï¸ 19.08.2020):

	```
	git clone https://github.com/lh3/bedtk
	```
</details>
<br>

## Tools for SAM/BAM

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/broadinstitute/picard">picard</a></b> (ğŸ¥‡22 Â·  â­ 640) - A set of command line tools (in Java) for manipulating high-throughput..</summary>

- [GitHub](https://github.com/broadinstitute/picard) (ğŸ‘¨â€ğŸ’» 140 Â· ğŸ”€ 290 Â· ğŸ“¥ 160K Â· ğŸ“‹ 630 - 26% open Â· â±ï¸ 03.02.2021):

	```
	git clone https://github.com/broadinstitute/picard
	```
</details>
<details><summary><b><a href="https://github.com/samtools/samtools">samtools</a></b> (ğŸ¥ˆ20 Â·  â­ 1K) - Tools (written in C using htslib) for manipulating next-generation sequencing..</summary>

- [GitHub](https://github.com/samtools/samtools) (ğŸ‘¨â€ğŸ’» 77 Â· ğŸ”€ 440 Â· ğŸ“¥ 780K Â· ğŸ“‹ 830 - 20% open Â· â±ï¸ 03.02.2021):

	```
	git clone https://github.com/samtools/samtools
	```
</details>
<details><summary><b><a href="https://github.com/samtools/htslib">htslib</a></b> (ğŸ¥ˆ19 Â·  â­ 520) - C library for high-throughput sequencing data formats.</summary>

- [GitHub](https://github.com/samtools/htslib) (ğŸ‘¨â€ğŸ’» 68 Â· ğŸ”€ 340 Â· ğŸ“¥ 370K Â· ğŸ“‹ 500 - 23% open Â· â±ï¸ 05.02.2021):

	```
	git clone https://github.com/samtools/htslib
	```
</details>
<details><summary><b><a href="https://github.com/biod/sambamba">sambamba</a></b> (ğŸ¥ˆ19 Â·  â­ 400) - Tools for working with SAM/BAM data.</summary>

- [GitHub](https://github.com/biod/sambamba) (ğŸ‘¨â€ğŸ’» 22 Â· ğŸ”€ 74 Â· ğŸ“¥ 100K Â· ğŸ“‹ 410 - 2% open Â· â±ï¸ 01.12.2020):

	```
	git clone https://github.com/biod/sambamba
	```
</details>
<details><summary><b><a href="https://github.com/CGATOxford/UMI-tools">UMI-tools</a></b> (ğŸ¥ˆ19 Â·  â­ 310) - Tools for handling Unique Molecular Identifiers in NGS data sets.</summary>

- [GitHub](https://github.com/CGATOxford/UMI-tools) (ğŸ‘¨â€ğŸ’» 29 Â· ğŸ”€ 160 Â· ğŸ“¥ 3.2K Â· ğŸ“‹ 320 - 8% open Â· â±ï¸ 05.02.2021):

	```
	git clone https://github.com/CGATOxford/UMI-tools
	```
</details>
<details><summary><b><a href="https://github.com/brentp/mosdepth">mosdepth</a></b> (ğŸ¥‰18 Â·  â­ 370) - fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing.</summary>

- [GitHub](https://github.com/brentp/mosdepth) (ğŸ‘¨â€ğŸ’» 14 Â· ğŸ”€ 65 Â· ğŸ“¥ 63K Â· ğŸ“‹ 120 - 17% open Â· â±ï¸ 20.11.2020):

	```
	git clone https://github.com/brentp/mosdepth
	```
</details>
<details><summary><b><a href="https://github.com/pysam-developers/pysam">pysam</a></b> (ğŸ¥‰17 Â·  â­ 480) - Pysam is a Python module for reading and manipulating SAM/BAM/VCF/BCF files. It's..</summary>

- [GitHub](https://github.com/pysam-developers/pysam) (ğŸ‘¨â€ğŸ’» 96 Â· ğŸ”€ 200 Â· ğŸ“‹ 690 - 28% open Â· â±ï¸ 11.01.2021):

	```
	git clone https://github.com/pysam-developers/pysam
	```
</details>
<details><summary><b><a href="https://github.com/GregoryFaust/samblaster">samblaster</a></b> (ğŸ¥‰13 Â·  â­ 160) - samblaster: a tool to mark duplicates and extract discordant and split..</summary>

- [GitHub](https://github.com/GregoryFaust/samblaster) (ğŸ‘¨â€ğŸ’» 5 Â· ğŸ”€ 21 Â· ğŸ“¥ 7.8K Â· ğŸ“‹ 42 - 19% open Â· â±ï¸ 04.06.2020):

	```
	git clone https://github.com/GregoryFaust/samblaster
	```
</details>
<details><summary><b><a href="https://github.com/gt1/biobambam2">biobambam2</a></b> (ğŸ¥‰13 Â·  â­ 84 Â· ğŸ’¤) - Tools for early stage alignment file processing.</summary>

- [GitHub](https://github.com/gt1/biobambam2) (ğŸ‘¨â€ğŸ’» 8 Â· ğŸ”€ 16 Â· ğŸ“¥ 11K Â· ğŸ“‹ 70 - 38% open Â· â±ï¸ 12.03.2019):

	```
	git clone https://github.com/gt1/biobambam2
	```
</details>
<details><summary><b><a href="https://github.com/samtools/hts-specs">hts-specs</a></b> (ğŸ¥‰12 Â·  â­ 420) - Specifications of SAM/BAM and related high-throughput sequencing file formats.</summary>

- [GitHub](https://github.com/samtools/hts-specs) (ğŸ‘¨â€ğŸ’» 50 Â· ğŸ”€ 140 Â· ğŸ“‹ 270 - 44% open Â· â±ï¸ 04.02.2021):

	```
	git clone https://github.com/samtools/hts-specs
	```
</details>
<br>

## Tools for VCF/BCF

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/samtools/bcftools">bcftools</a></b> (ğŸ¥‡19 Â·  â­ 360) - This is the official development repository for BCFtools. To compile, the..</summary>

- [GitHub](https://github.com/samtools/bcftools) (ğŸ‘¨â€ğŸ’» 45 Â· ğŸ”€ 160 Â· ğŸ“¥ 280K Â· ğŸ“‹ 1.2K - 15% open Â· â±ï¸ 05.02.2021):

	```
	git clone https://github.com/samtools/bcftools
	```
</details>
<details><summary><b><a href="https://github.com/vcflib/vcflib">vcflib</a></b> (ğŸ¥ˆ18 Â·  â­ 410) - C++ library and cmdline tools for parsing and manipulating VCF files.</summary>

- [GitHub](https://github.com/vcflib/vcflib) (ğŸ‘¨â€ğŸ’» 66 Â· ğŸ”€ 180 Â· ğŸ“¥ 820 Â· ğŸ“‹ 210 - 9% open Â· â±ï¸ 30.01.2021):

	```
	git clone https://github.com/vcflib/vcflib
	```
</details>
<details><summary><b><a href="https://github.com/brentp/cyvcf2">cyvcf2</a></b> (ğŸ¥‰17 Â·  â­ 240) - cython + htslib == fast VCF and BCF processing.</summary>

- [GitHub](https://github.com/brentp/cyvcf2) (ğŸ‘¨â€ğŸ’» 29 Â· ğŸ”€ 48 Â· ğŸ“¦ 180 Â· ğŸ“‹ 140 - 20% open Â· â±ï¸ 26.01.2021):

	```
	git clone https://github.com/brentp/cyvcf2
	```
</details>
<details><summary><b><a href="https://github.com/brentp/vcfanno">vcfanno</a></b> (ğŸ¥‰16 Â·  â­ 260 Â· â•) - annotate a VCF with other VCFs/BEDs/tabixed files.</summary>

- [GitHub](https://github.com/brentp/vcfanno) (ğŸ‘¨â€ğŸ’» 9 Â· ğŸ”€ 44 Â· ğŸ“¥ 22K Â· ğŸ“‹ 120 - 21% open Â· â±ï¸ 15.04.2020):

	```
	git clone https://github.com/brentp/vcfanno
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/hap.py">hap.py</a></b> (ğŸ¥‰11 Â·  â­ 250) - Haplotype VCF comparison tools.</summary>

- [GitHub](https://github.com/Illumina/hap.py) (ğŸ‘¨â€ğŸ’» 18 Â· ğŸ”€ 77 Â· ğŸ“¥ 50 Â· ğŸ“‹ 120 - 56% open Â· â±ï¸ 13.02.2020):

	```
	git clone https://github.com/Illumina/hap.py
	```
</details>
<br>

## Assembler

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/ruanjue/wtdbg2">wtdbg2</a></b> (ğŸ¥‡14 Â·  â­ 400) - Redbean: A fuzzy Bruijn graph approach to long noisy reads assembly.</summary>

- [GitHub](https://github.com/ruanjue/wtdbg2) (ğŸ‘¨â€ğŸ’» 7 Â· ğŸ”€ 79 Â· ğŸ“¥ 1.3K Â· ğŸ“‹ 210 - 6% open Â· â±ï¸ 11.12.2020):

	```
	git clone https://github.com/ruanjue/wtdbg2
	```
</details>
<details><summary><b><a href="https://github.com/aquaskyline/SOAPdenovo2">SOAPdenovo2</a></b> (ğŸ¥‰13 Â·  â­ 150) - Next generation sequencing reads de novo assembler.</summary>

- [GitHub](https://github.com/aquaskyline/SOAPdenovo2) (ğŸ‘¨â€ğŸ’» 11 Â· ğŸ”€ 51 Â· ğŸ“‹ 60 - 1% open Â· â±ï¸ 21.01.2021):

	```
	git clone https://github.com/aquaskyline/SOAPdenovo2
	```
</details>
<details><summary><b><a href="https://github.com/dzerbino/velvet">velvet</a></b> (ğŸ¥‰12 Â·  â­ 230 Â· ğŸ’€) - Short read de novo assembler using de Bruijn graphs.</summary>

- [GitHub](https://github.com/dzerbino/velvet) (ğŸ‘¨â€ğŸ’» 15 Â· ğŸ”€ 84 Â· â±ï¸ 15.08.2014):

	```
	git clone https://github.com/dzerbino/velvet
	```
</details>
<br>

## NGS Alignment

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/lh3/minimap2">minimap2</a></b> (ğŸ¥‡19 Â·  â­ 890) - A versatile pairwise aligner for genomic and spliced nucleotide sequences.</summary>

- [GitHub](https://github.com/lh3/minimap2) (ğŸ‘¨â€ğŸ’» 28 Â· ğŸ”€ 240 Â· ğŸ“¥ 41K Â· ğŸ“¦ 71 Â· ğŸ“‹ 650 - 8% open Â· â±ï¸ 07.02.2021):

	```
	git clone https://github.com/lh3/minimap2
	```
</details>
<details><summary><b><a href="https://github.com/BenLangmead/bowtie2">bowtie2</a></b> (ğŸ¥‡19 Â·  â­ 360) - A fast and sensitive gapped read aligner.</summary>

- [GitHub](https://github.com/BenLangmead/bowtie2) (ğŸ‘¨â€ğŸ’» 32 Â· ğŸ”€ 120 Â· ğŸ“¥ 36K Â· ğŸ“‹ 290 - 29% open Â· â±ï¸ 03.01.2021):

	```
	git clone https://github.com/BenLangmead/bowtie2
	```
</details>
<details><summary><b><a href="https://github.com/lh3/bwa">bwa</a></b> (ğŸ¥ˆ18 Â·  â­ 950) - Burrow-Wheeler Aligner for short-read alignment (see minimap2 for long-read..</summary>

- [GitHub](https://github.com/lh3/bwa) (ğŸ‘¨â€ğŸ’» 31 Â· ğŸ”€ 430 Â· ğŸ“¥ 50K Â· ğŸ“‹ 210 - 60% open Â· â±ï¸ 02.07.2020):

	```
	git clone https://github.com/lh3/bwa
	```
</details>
<details><summary><b><a href="https://github.com/alexdobin/STAR">STAR</a></b> (ğŸ¥ˆ17 Â·  â­ 1K) - RNA-seq aligner.</summary>

- [GitHub](https://github.com/alexdobin/STAR) (ğŸ‘¨â€ğŸ’» 36 Â· ğŸ”€ 290 Â· ğŸ“¥ 350 Â· ğŸ“‹ 1.1K - 25% open Â· â±ï¸ 28.12.2020):

	```
	git clone https://github.com/alexdobin/STAR
	```
</details>
<details><summary><b><a href="https://github.com/DaehwanKimLab/hisat2">hisat2</a></b> (ğŸ¥‰16 Â·  â­ 280) - Graph-based alignment (Hierarchical Graph FM index).</summary>

- [GitHub](https://github.com/DaehwanKimLab/hisat2) (ğŸ‘¨â€ğŸ’» 26 Â· ğŸ”€ 74 Â· ğŸ“‹ 240 - 55% open Â· â±ï¸ 21.12.2020):

	```
	git clone https://github.com/DaehwanKimLab/hisat2
	```
</details>
<details><summary><b><a href="https://github.com/bwa-mem2/bwa-mem2">bwa-mem2</a></b> (ğŸ¥‰15 Â·  â­ 390) - The next version of bwa-mem.</summary>

- [GitHub](https://github.com/bwa-mem2/bwa-mem2) (ğŸ‘¨â€ğŸ’» 9 Â· ğŸ”€ 46 Â· ğŸ“¥ 5.2K Â· ğŸ“‹ 100 - 13% open Â· â±ï¸ 05.01.2021):

	```
	git clone https://github.com/bwa-mem2/bwa-mem2
	```
</details>
<details><summary><b><a href="https://github.com/philres/ngmlr">ngmlr</a></b> (ğŸ¥‰15 Â·  â­ 210 Â· ğŸ’¤) - NGMLR is a long-read mapper designed to align PacBio or Oxford Nanopore..</summary>

- [GitHub](https://github.com/philres/ngmlr) (ğŸ‘¨â€ğŸ’» 13 Â· ğŸ”€ 32 Â· ğŸ“¥ 3.9K Â· ğŸ“‹ 83 - 38% open Â· â±ï¸ 11.09.2018):

	```
	git clone https://github.com/philres/ngmlr
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/blasr">blasr</a></b> (ğŸ¥‰13 Â·  â­ 120) - BLASR: The PacBio long read aligner.</summary>

- [GitHub](https://github.com/PacificBiosciences/blasr) (ğŸ‘¨â€ğŸ’» 19 Â· ğŸ”€ 76 Â· â±ï¸ 16.12.2020):

	```
	git clone https://github.com/PacificBiosciences/blasr
	```
</details>
<details><summary><b><a href="https://github.com/DaehwanKimLab/tophat">tophat</a></b> (ğŸ¥‰12 Â·  â­ 87 Â· ğŸ’¤) - Spliced read mapper for RNA-Seq.</summary>

- [GitHub](https://github.com/DaehwanKimLab/tophat) (ğŸ‘¨â€ğŸ’» 25 Â· ğŸ”€ 35 Â· ğŸ“‹ 46 - 76% open Â· â±ï¸ 24.05.2018):

	```
	git clone https://github.com/DaehwanKimLab/tophat
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/pbmm2">pbmm2</a></b> (ğŸ¥‰10 Â·  â­ 75) - A minimap2 frontend for PacBio native data formats.</summary>

- [GitHub](https://github.com/PacificBiosciences/pbmm2) (ğŸ‘¨â€ğŸ’» 10 Â· ğŸ”€ 15 Â· â±ï¸ 22.09.2020):

	```
	git clone https://github.com/PacificBiosciences/pbmm2
	```
</details>
<br>

## Variant Caller

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/broadinstitute/gatk">gatk</a></b> (ğŸ¥‡21 Â·  â­ 980) - Official code repository for GATK versions 4 and up.</summary>

- [GitHub](https://github.com/broadinstitute/gatk) (ğŸ‘¨â€ğŸ’» 110 Â· ğŸ”€ 380 Â· ğŸ“¥ 330K Â· ğŸ“‹ 4K - 23% open Â· â±ï¸ 03.02.2021):

	```
	git clone https://github.com/broadinstitute/gatk
	```
</details>
<details><summary><b><a href="https://github.com/google/deepvariant">deepvariant</a></b> (ğŸ¥ˆ20 Â·  â­ 2.2K) - DeepVariant is an analysis pipeline that uses a deep neural network to..</summary>

- [GitHub](https://github.com/google/deepvariant) (ğŸ‘¨â€ğŸ’» 19 Â· ğŸ”€ 530 Â· ğŸ“¥ 3K Â· ğŸ“‹ 380 - 0% open Â· â±ï¸ 14.12.2020):

	```
	git clone https://github.com/google/deepvariant
	```
</details>
<details><summary><b><a href="https://github.com/freebayes/freebayes">freebayes</a></b> (ğŸ¥ˆ20 Â·  â­ 510) - Bayesian haplotype-based genetic polymorphism discovery and genotyping.</summary>

- [GitHub](https://github.com/freebayes/freebayes) (ğŸ‘¨â€ğŸ’» 42 Â· ğŸ”€ 210 Â· ğŸ“¥ 3.7K Â· ğŸ“‹ 570 - 0% open Â· â±ï¸ 06.02.2021):

	```
	git clone https://github.com/freebayes/freebayes
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/strelka">strelka</a></b> (ğŸ¥‰16 Â·  â­ 240 Â· ğŸ’¤) - Strelka2 germline and somatic small variant caller.</summary>

- [GitHub](https://github.com/Illumina/strelka) (ğŸ‘¨â€ğŸ’» 28 Â· ğŸ”€ 68 Â· ğŸ“¥ 15K Â· ğŸ“‹ 190 - 44% open Â· â±ï¸ 06.02.2019):

	```
	git clone https://github.com/Illumina/strelka
	```
</details>
<details><summary><b><a href="https://github.com/AstraZeneca-NGS/VarDict">VarDict</a></b> (ğŸ¥‰13 Â·  â­ 140) - VarDict.</summary>

- [GitHub](https://github.com/AstraZeneca-NGS/VarDict) (ğŸ‘¨â€ğŸ’» 21 Â· ğŸ”€ 49 Â· ğŸ“‹ 92 - 15% open Â· â±ï¸ 25.01.2021):

	```
	git clone https://github.com/AstraZeneca-NGS/VarDict
	```
</details>
<details><summary><b><a href="https://github.com/bioinform/neusomatic">neusomatic</a></b> (ğŸ¥‰13 Â·  â­ 120) - NeuSomatic: Deep convolutional neural networks for accurate somatic..</summary>

- [GitHub](https://github.com/bioinform/neusomatic) (ğŸ‘¨â€ğŸ’» 25 Â· ğŸ”€ 37 Â· ğŸ“‹ 39 - 33% open Â· â±ï¸ 28.02.2020):

	```
	git clone https://github.com/bioinform/neusomatic
	```
</details>
<details><summary><b><a href="https://github.com/dkoboldt/varscan">varscan</a></b> (ğŸ¥‰10 Â·  â­ 120 Â· ğŸ’¤) - Variant calling and somatic mutation/CNV detection for next-generation..</summary>

- [GitHub](https://github.com/dkoboldt/varscan) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 29 Â· ğŸ“¥ 12K Â· ğŸ“‹ 60 - 90% open Â· â±ï¸ 25.07.2019):

	```
	git clone https://github.com/dkoboldt/varscan
	```
</details>
<details><summary><b><a href="https://github.com/broadinstitute/mutect">mutect</a></b> (ğŸ¥‰10 Â·  â­ 87 Â· ğŸ’€) - MuTect -- Accurate and sensitive cancer mutation detection.</summary>

- [GitHub](https://github.com/broadinstitute/mutect) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 40 Â· ğŸ“¥ 5.8K Â· ğŸ“‹ 9 - 88% open Â· â±ï¸ 01.06.2014):

	```
	git clone https://github.com/broadinstitute/mutect
	```
</details>
<br>

## Variant Annotation

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/Ensembl/ensembl-vep">ensembl-vep</a></b> (ğŸ¥‡17 Â·  â­ 250) - The Ensembl Variant Effect Predictor predicts the functional effects of..</summary>

- [GitHub](https://github.com/Ensembl/ensembl-vep) (ğŸ‘¨â€ğŸ’» 22 Â· ğŸ”€ 94 Â· ğŸ“‹ 540 - 14% open Â· â±ï¸ 10.11.2020):

	```
	git clone https://github.com/Ensembl/ensembl-vep
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/SpliceAI">SpliceAI</a></b> (ğŸ¥ˆ12 Â·  â­ 160) - A deep learning-based tool to identify splice variants.</summary>

- [GitHub](https://github.com/Illumina/SpliceAI) (ğŸ‘¨â€ğŸ’» 11 Â· ğŸ”€ 66 Â· ğŸ“¦ 4 Â· â±ï¸ 15.05.2020):

	```
	git clone https://github.com/Illumina/SpliceAI
	```
</details>
<details><summary><b><a href="https://github.com/pcingola/SnpEff">SnpEff</a></b> (ğŸ¥‰10 Â·  â­ 110) - </summary>

- [GitHub](https://github.com/pcingola/SnpEff) (ğŸ‘¨â€ğŸ’» 19 Â· ğŸ”€ 51 Â· ğŸ“‹ 260 - 4% open Â· â±ï¸ 31.01.2021):

	```
	git clone https://github.com/pcingola/SnpEff
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/InterVar">InterVar</a></b> (ğŸ¥‰10 Â·  â­ 110) - A bioinformatics software tool for clinical interpretation of genetic variants..</summary>

- [GitHub](https://github.com/WGLab/InterVar) (ğŸ‘¨â€ğŸ’» 3 Â· ğŸ”€ 61 Â· ğŸ“‹ 52 - 48% open Â· â±ï¸ 05.02.2021):

	```
	git clone https://github.com/WGLab/InterVar
	```
</details>
<br>

## Structural Variation

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/dellytools/delly">delly</a></b> (ğŸ¥‡19 Â·  â­ 240) - DELLY2: Structural variant discovery by integrated paired-end and split-read..</summary>

- [GitHub](https://github.com/dellytools/delly) (ğŸ‘¨â€ğŸ’» 9 Â· ğŸ”€ 98 Â· ğŸ“¥ 17K Â· ğŸ“‹ 210 - 5% open Â· â±ï¸ 07.02.2021):

	```
	git clone https://github.com/dellytools/delly
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/manta">manta</a></b> (ğŸ¥ˆ16 Â·  â­ 260 Â· ğŸ’¤) - Structural variant and indel caller for mapped sequencing data.</summary>

- [GitHub](https://github.com/Illumina/manta) (ğŸ‘¨â€ğŸ’» 22 Â· ğŸ”€ 84 Â· ğŸ“¥ 22K Â· ğŸ“‹ 250 - 33% open Â· â±ï¸ 17.07.2019):

	```
	git clone https://github.com/Illumina/manta
	```
</details>
<details><summary><b><a href="https://github.com/etal/cnvkit">cnvkit</a></b> (ğŸ¥ˆ15 Â·  â­ 280) - Copy number variant detection from targeted DNA sequencing.</summary>

- [GitHub](https://github.com/etal/cnvkit) (ğŸ‘¨â€ğŸ’» 22 Â· ğŸ”€ 110 Â· ğŸ“‹ 500 - 28% open Â· â±ï¸ 14.12.2020):

	```
	git clone https://github.com/etal/cnvkit
	```
</details>
<details><summary><b><a href="https://github.com/brentp/smoove">smoove</a></b> (ğŸ¥ˆ15 Â·  â­ 140 Â· â•) - structural variant calling and genotyping with existing tools, but, smoothly.</summary>

- [GitHub](https://github.com/brentp/smoove) (ğŸ‘¨â€ğŸ’» 5 Â· ğŸ”€ 12 Â· ğŸ“¥ 2.9K Â· ğŸ“‹ 130 - 32% open Â· â±ï¸ 29.01.2021):

	```
	git clone https://github.com/brentp/smoove
	```
</details>
<details><summary><b><a href="https://github.com/fritzsedlazeck/Sniffles">Sniffles</a></b> (ğŸ¥‰14 Â·  â­ 240) - Structural variation caller using third generation sequencing.</summary>

- [GitHub](https://github.com/fritzsedlazeck/Sniffles) (ğŸ‘¨â€ğŸ’» 10 Â· ğŸ”€ 45 Â· ğŸ“‹ 240 - 17% open Â· â±ï¸ 18.12.2020):

	```
	git clone https://github.com/fritzsedlazeck/Sniffles
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/GeneFuse">GeneFuse</a></b> (ğŸ¥‰13 Â·  â­ 67) - Gene fusion detection and visualization.</summary>

- [GitHub](https://github.com/OpenGene/GeneFuse) (ğŸ‘¨â€ğŸ’» 3 Â· ğŸ”€ 45 Â· ğŸ“‹ 27 - 88% open Â· â±ï¸ 17.09.2020):

	```
	git clone https://github.com/OpenGene/GeneFuse
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/RepeatHMM">RepeatHMM</a></b> (ğŸ¥‰12 Â·  â­ 22) - a hidden Markov model to infer simple repeats from genome sequences.</summary>

- [GitHub](https://github.com/WGLab/RepeatHMM) (ğŸ‘¨â€ğŸ’» 9 Â· ğŸ”€ 12 Â· ğŸ“¥ 36 Â· ğŸ“‹ 33 - 12% open Â· â±ï¸ 30.12.2020):

	```
	git clone https://github.com/WGLab/RepeatHMM
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/pbsv">pbsv</a></b> (ğŸ¥‰11 Â·  â­ 48) - pbsv - PacBio structural variant (SV) calling and analysis tools.</summary>

- [GitHub](https://github.com/PacificBiosciences/pbsv) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 8 Â· â±ï¸ 15.10.2020):

	```
	git clone https://github.com/PacificBiosciences/pbsv
	```
</details>
<details><summary><b><a href="https://github.com/lgmgeo/AnnotSV">AnnotSV</a></b> (ğŸ¥‰8 Â·  â­ 57) - Annotation and Ranking of Structural Variation.</summary>

- [GitHub](https://github.com/lgmgeo/AnnotSV) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 11 Â· ğŸ“‹ 36 - 8% open Â· â±ï¸ 19.01.2021):

	```
	git clone https://github.com/lgmgeo/AnnotSV
	```
</details>
<br>

## MSI Detection

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/ding-lab/msisensor">msisensor</a></b> (ğŸ¥‡12 Â·  â­ 100) - microsatellite instability detection using tumor only or paired tumor-normal..</summary>

- [GitHub](https://github.com/ding-lab/msisensor) (ğŸ‘¨â€ğŸ’» 5 Â· ğŸ”€ 45 Â· ğŸ“¥ 340 Â· ğŸ“‹ 42 - 23% open Â· â±ï¸ 06.01.2021):

	```
	git clone https://github.com/ding-lab/msisensor
	```
</details>
<details><summary><b><a href="https://github.com/xjtu-omics/msisensor-pro">msisensor-pro</a></b> (ğŸ¥ˆ10 Â·  â­ 24) - Microsatellite Instability (MSI) detection using high-throughput..</summary>

- [GitHub](https://github.com/xjtu-omics/msisensor-pro) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 7 Â· ğŸ“¥ 1 Â· ğŸ“‹ 18 - 44% open Â· â±ï¸ 03.02.2021):

	```
	git clone https://github.com/xjtu-omics/msisensor-pro
	```
</details>
<details><summary><b><a href="https://github.com/niu-lab/msisensor2">msisensor2</a></b> (ğŸ¥‰8 Â·  â­ 51 Â· ğŸ’¤) - Microsatellite instability (MSI) detection for tumor only data.</summary>

- [GitHub](https://github.com/niu-lab/msisensor2) (ğŸ‘¨â€ğŸ’» 3 Â· ğŸ”€ 12 Â· ğŸ“‹ 19 - 84% open Â· â±ï¸ 21.11.2019):

	```
	git clone https://github.com/niu-lab/msisensor2
	```
</details>
<details><summary><b><a href="https://github.com/niu-lab/msisensor-ct">msisensor-ct</a></b> (ğŸ¥‰6 Â·  â­ 4 Â· ğŸ£) - Microsatellite instability (MSI) detection for cfDNA samples.</summary>

- [GitHub](https://github.com/niu-lab/msisensor-ct) (ğŸ‘¨â€ğŸ’» 2 Â· â±ï¸ 29.01.2021):

	```
	git clone https://github.com/niu-lab/msisensor-ct
	```
</details>
<br>

## MHC/HLA

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/openvax/mhcflurry">mhcflurry</a></b> (ğŸ¥‡19 Â·  â­ 110) - Peptide-MHC I binding affinity prediction.</summary>

- [GitHub](https://github.com/openvax/mhcflurry) (ğŸ‘¨â€ğŸ’» 12 Â· ğŸ”€ 32 Â· ğŸ“¥ 43K Â· ğŸ“¦ 14 Â· ğŸ“‹ 100 - 1% open Â· â±ï¸ 07.01.2021):

	```
	git clone https://github.com/openvax/mhcflurry
	```
</details>
<details><summary><b><a href="https://github.com/FRED-2/OptiType">OptiType</a></b> (ğŸ¥ˆ13 Â·  â­ 110) - Precision HLA typing from next-generation sequencing data.</summary>

- [GitHub](https://github.com/FRED-2/OptiType) (ğŸ‘¨â€ğŸ’» 13 Â· ğŸ”€ 63 Â· ğŸ“‹ 88 - 31% open Â· â±ï¸ 15.09.2020):

	```
	git clone https://github.com/FRED-2/OptiType
	```
</details>
<details><summary><b><a href="https://github.com/ANHIG/IMGTHLA">IMGTHLA</a></b> (ğŸ¥‰9 Â·  â­ 81) - Github for files currently published in the IPD-IMGT/HLA FTP Directory hosted at..</summary>

- [GitHub](https://github.com/ANHIG/IMGTHLA) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 35 Â· â±ï¸ 04.02.2021):

	```
	git clone https://github.com/ANHIG/IMGTHLA
	```
</details>
<details><summary><b><a href="https://github.com/humanlongevity/HLA">HLA</a></b> (ğŸ¥‰9 Â·  â­ 66 Â· ğŸ’€) - xHLA: Fast and accurate HLA typing from short read sequence data.</summary>

- [GitHub](https://github.com/humanlongevity/HLA) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 34 Â· ğŸ“‹ 55 - 70% open Â· â±ï¸ 04.10.2017):

	```
	git clone https://github.com/humanlongevity/HLA
	```
</details>
<br>

## Sequence Alignment

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/bbuchfink/diamond">diamond</a></b> (ğŸ¥‡19 Â·  â­ 460) - Accelerated BLAST compatible local sequence aligner.</summary>

- [GitHub](https://github.com/bbuchfink/diamond) (ğŸ‘¨â€ğŸ’» 17 Â· ğŸ”€ 120 Â· ğŸ“¥ 230K Â· ğŸ“‹ 400 - 10% open Â· â±ï¸ 27.01.2021):

	```
	git clone https://github.com/bbuchfink/diamond
	```
</details>
<details><summary><b><a href="https://github.com/icebert/pblat">pblat</a></b> (ğŸ¥ˆ8 Â·  â­ 32) - parallelized blat with multi-threads support.</summary>

- [GitHub](https://github.com/icebert/pblat) (ğŸ‘¨â€ğŸ’» 5 Â· ğŸ”€ 9 Â· ğŸ“‹ 14 - 21% open Â· â±ï¸ 14.06.2020):

	```
	git clone https://github.com/icebert/pblat
	```
</details>
<details><summary><b><a href="https://github.com/djhshih/blat">blat</a></b> (ğŸ¥ˆ6 Â·  â­ 10 Â· ğŸ’¤) - BLAST-Like Alignment Tool.</summary>

- [GitHub](https://github.com/djhshih/blat) (ğŸ”€ 5 Â· â±ï¸ 11.01.2019):

	```
	git clone https://github.com/djhshih/blat
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/clustal-omega">clustal-omega</a></b> (ğŸ¥ˆ6 Â·  â­ 4 Â· ğŸ’€) - Multiple sequence aligner for protein and DNA/RNA.</summary>

- [GitHub](https://github.com/GSLBiotech/clustal-omega) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 1 Â· â±ï¸ 02.01.2018):

	```
	git clone https://github.com/GSLBiotech/clustal-omega
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/t-coffee">t-coffee</a></b> (ğŸ¥‰5 Â·  â­ 2) - Tools for multiple alignments of DNA, RNA, and Proteins.</summary>

- [GitHub](https://github.com/GSLBiotech/t-coffee) (ğŸ‘¨â€ğŸ’» 5 Â· â±ï¸ 12.02.2020):

	```
	git clone https://github.com/GSLBiotech/t-coffee
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/mafft">mafft</a></b> (ğŸ¥‰4 Â·  â­ 9) - Align multiple amino acid or nucleotide sequences.</summary>

- [GitHub](https://github.com/GSLBiotech/mafft) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 3 Â· â±ï¸ 17.09.2020):

	```
	git clone https://github.com/GSLBiotech/mafft
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/muscle">muscle</a></b> (ğŸ¥‰3 Â·  â­ 2) - Multiple Sequence Comparison by Log-Expectation.</summary>

- [GitHub](https://github.com/GSLBiotech/muscle) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 1 Â· â±ï¸ 07.10.2020):

	```
	git clone https://github.com/GSLBiotech/muscle
	```
</details>
<br>

## Visualization

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/lmcinnes/umap">umap</a></b> (ğŸ¥‡25 Â·  â­ 4.5K) - Uniform Manifold Approximation and Projection.</summary>

- [GitHub](https://github.com/lmcinnes/umap) (ğŸ‘¨â€ğŸ’» 71 Â· ğŸ”€ 470 Â· ğŸ“¦ 2.3K Â· ğŸ“‹ 430 - 47% open Â· â±ï¸ 06.02.2021):

	```
	git clone https://github.com/lmcinnes/umap
	```
</details>
<details><summary><b><a href="https://github.com/igvteam/igv.js">igv.js</a></b> (ğŸ¥ˆ21 Â·  â­ 400) - Embeddable genomic visualization component based on the Integrative Genomics..</summary>

- [GitHub](https://github.com/igvteam/igv.js) (ğŸ‘¨â€ğŸ’» 50 Â· ğŸ”€ 140 Â· ğŸ“¦ 85 Â· ğŸ“‹ 810 - 6% open Â· â±ï¸ 12.01.2021):

	```
	git clone https://github.com/igvteam/igv.js
	```
</details>
<details><summary><b><a href="https://github.com/igvteam/igv">igv</a></b> (ğŸ¥‰16 Â·  â­ 420) - Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for..</summary>

- [GitHub](https://github.com/igvteam/igv) (ğŸ‘¨â€ğŸ’» 53 Â· ğŸ”€ 190 Â· ğŸ“‹ 720 - 7% open Â· â±ï¸ 05.02.2021):

	```
	git clone https://github.com/igvteam/igv
	```
</details>
<details><summary><b><a href="https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer">DnaFeaturesViewer</a></b> (ğŸ¥‰16 Â·  â­ 250) - Python library to plot DNA sequence features (e.g. from Genbank..</summary>

- [GitHub](https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer) (ğŸ‘¨â€ğŸ’» 5 Â· ğŸ”€ 43 Â· ğŸ“‹ 46 - 47% open Â· â±ï¸ 20.09.2020):

	```
	git clone https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer
	```
</details>
<details><summary><b><a href="https://github.com/deeptools/pyGenomeTracks">pyGenomeTracks</a></b> (ğŸ¥‰15 Â·  â­ 430) - python module to plot beautiful and highly customizable genome browser..</summary>

- [GitHub](https://github.com/deeptools/pyGenomeTracks) (ğŸ‘¨â€ğŸ’» 11 Â· ğŸ”€ 70 Â· ğŸ“¦ 4 Â· ğŸ“‹ 120 - 10% open Â· â±ï¸ 12.11.2020):

	```
	git clone https://github.com/deeptools/pyGenomeTracks
	```
</details>
<br>

## Others

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/biopython/biopython">biopython</a></b> (ğŸ¥‡24 Â·  â­ 2.6K) - Official git repository for Biopython (originally converted from CVS).</summary>

- [GitHub](https://github.com/biopython/biopython) (ğŸ‘¨â€ğŸ’» 370 Â· ğŸ”€ 1.2K Â· ğŸ“¦ 6.5K Â· ğŸ“‹ 1.2K - 27% open Â· â±ï¸ 07.02.2021):

	```
	git clone https://github.com/biopython/biopython
	```
</details>
<details><summary><b><a href="https://github.com/bioconda/bioconda-recipes">bioconda-recipes</a></b> (ğŸ¥‡19 Â·  â­ 1.2K) - Conda recipes for the bioconda channel.</summary>

- [GitHub](https://github.com/bioconda/bioconda-recipes) (ğŸ‘¨â€ğŸ’» 1.3K Â· ğŸ”€ 1.7K Â· ğŸ“‹ 1.4K - 18% open Â· â±ï¸ 07.02.2021):

	```
	git clone https://github.com/bioconda/bioconda-recipes
	```
</details>
<details><summary><b><a href="https://github.com/ewels/MultiQC">MultiQC</a></b> (ğŸ¥‡19 Â·  â­ 700) - Aggregate results from bioinformatics analyses across many samples into a..</summary>

- [GitHub](https://github.com/ewels/MultiQC) (ğŸ‘¨â€ğŸ’» 140 Â· ğŸ”€ 350 Â· ğŸ“¥ 43 Â· ğŸ“¦ 130 Â· ğŸ“‹ 920 - 16% open Â· â±ï¸ 03.02.2021):

	```
	git clone https://github.com/ewels/MultiQC
	```
</details>
<details><summary><b><a href="https://github.com/Gaius-Augustus/Augustus">Augustus</a></b> (ğŸ¥ˆ16 Â·  â­ 120) - Genome annotation with AUGUSTUS.</summary>

- [GitHub](https://github.com/Gaius-Augustus/Augustus) (ğŸ‘¨â€ğŸ’» 30 Â· ğŸ”€ 60 Â· ğŸ“¥ 2.5K Â· ğŸ“‹ 170 - 56% open Â· â±ï¸ 28.01.2021):

	```
	git clone https://github.com/Gaius-Augustus/Augustus
	```
</details>
<details><summary><b><a href="https://github.com/macs3-project/MACS">MACS</a></b> (ğŸ¥ˆ15 Â·  â­ 470) - MACS -- Model-based Analysis of ChIP-Seq.</summary>

- [GitHub](https://github.com/macs3-project/MACS) (ğŸ‘¨â€ğŸ’» 18 Â· ğŸ”€ 210 Â· ğŸ“‹ 370 - 51% open Â· â±ï¸ 05.02.2021):

	```
	git clone https://github.com/macs3-project/MACS
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/Phen2Gene">Phen2Gene</a></b> (ğŸ¥ˆ13 Â·  â­ 23) - Phenotype driven gene prioritization for HPO.</summary>

- [GitHub](https://github.com/WGLab/Phen2Gene) (ğŸ‘¨â€ğŸ’» 14 Â· ğŸ”€ 8 Â· ğŸ“¥ 650 Â· ğŸ“‹ 8 - 25% open Â· â±ï¸ 12.01.2021):

	```
	git clone https://github.com/WGLab/Phen2Gene
	```
</details>
<details><summary><b><a href="https://github.com/EddyRivasLab/hmmer">hmmer</a></b> (ğŸ¥‰12 Â·  â­ 150) - HMMER: biological sequence analysis using profile HMMs.</summary>

- [GitHub](https://github.com/EddyRivasLab/hmmer) (ğŸ‘¨â€ğŸ’» 18 Â· ğŸ”€ 41 Â· ğŸ“‹ 190 - 3% open Â· â±ï¸ 27.11.2020):

	```
	git clone https://github.com/EddyRivasLab/hmmer
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/GTCtoVCF">GTCtoVCF</a></b> (ğŸ¥‰11 Â·  â­ 25) - Script to convert GTC/BPM files to VCF.</summary>

- [GitHub](https://github.com/Illumina/GTCtoVCF) (ğŸ‘¨â€ğŸ’» 10 Â· ğŸ”€ 18 Â· ğŸ“‹ 38 - 26% open Â· â±ï¸ 07.04.2020):

	```
	git clone https://github.com/Illumina/GTCtoVCF
	```
</details>
<details><summary><b><a href="https://github.com/liguowang/CrossMap">CrossMap</a></b> (ğŸ¥‰11 Â·  â­ 12) - CrossMap is a python program to lift over genome coordinates from one genome..</summary>

- [GitHub](https://github.com/liguowang/CrossMap) (ğŸ‘¨â€ğŸ’» 2 Â· ğŸ”€ 10 Â· ğŸ“¦ 11 Â· ğŸ“‹ 22 - 59% open Â· â±ï¸ 14.01.2021):

	```
	git clone https://github.com/liguowang/CrossMap
	```
</details>
<details><summary><b><a href="https://github.com/mutalyzer/mutalyzer">mutalyzer</a></b> (ğŸ¥‰10 Â·  â­ 79) - HGVS variant nomenclature checker.</summary>

- [GitHub](https://github.com/mutalyzer/mutalyzer) (ğŸ‘¨â€ğŸ’» 9 Â· ğŸ”€ 18 Â· ğŸ“‹ 320 - 38% open Â· â±ï¸ 24.11.2020):

	```
	git clone https://github.com/mutalyzer/mutalyzer
	```
</details>
<details><summary><b><a href="https://github.com/PharmGKB/PharmCAT">PharmCAT</a></b> (ğŸ¥‰10 Â·  â­ 38) - The Pharmacogenomic Clinical Annotation Tool.</summary>

- [GitHub](https://github.com/PharmGKB/PharmCAT) (ğŸ‘¨â€ğŸ’» 15 Â· ğŸ”€ 16 Â· ğŸ“¥ 1K Â· ğŸ“‹ 34 - 20% open Â· â±ï¸ 09.09.2020):

	```
	git clone https://github.com/PharmGKB/PharmCAT
	```
</details>
<details><summary><b><a href="https://github.com/samtools/tabix">tabix</a></b> (ğŸ¥‰9 Â·  â­ 80 Â· ğŸ’€) - Note: tabix and bgzip binaries are now part of the HTSlib project.</summary>

- [GitHub](https://github.com/samtools/tabix) (ğŸ‘¨â€ğŸ’» 4 Â· ğŸ”€ 37 Â· ğŸ“‹ 13 - 53% open Â· â±ï¸ 16.12.2013):

	```
	git clone https://github.com/samtools/tabix
	```
</details>
<details><summary><b><a href="https://github.com/rmhubley/RepeatMasker">RepeatMasker</a></b> (ğŸ¥‰9 Â·  â­ 69) - RepeatMasker is a program that screens DNA sequences for interspersed..</summary>

- [GitHub](https://github.com/rmhubley/RepeatMasker) (ğŸ‘¨â€ğŸ’» 3 Â· ğŸ”€ 25 Â· ğŸ“‹ 86 - 31% open Â· â±ï¸ 02.09.2020):

	```
	git clone https://github.com/rmhubley/RepeatMasker
	```
</details>
<details><summary><b><a href="https://github.com/lh3/tabtk">tabtk</a></b> (ğŸ¥‰6 Â·  â­ 53 Â· ğŸ’€) - Toolkit for processing TAB-delimited format.</summary>

- [GitHub](https://github.com/lh3/tabtk) (ğŸ”€ 12 Â· â±ï¸ 11.08.2016):

	```
	git clone https://github.com/lh3/tabtk
	```
</details>
